go.mod

```diff
@@ -25,7 +25,7 @@ require (
 	github.com/overmindtech/discovery v0.33.4
 	github.com/overmindtech/k8s-source v0.9.1
 	github.com/overmindtech/pterm v0.0.0-20240919144758-04d94ccb2297
-	github.com/overmindtech/sdp-go v0.102.4
+	github.com/overmindtech/sdp-go v0.102.5-0.20241216105318-a0af4432d4b2
```

Comment:
Once sdp-go is released this will be replaced: https://github.com/overmindtech/sdp-go/pull/348